### DISCUSSION TITLE
VPS

### DISCUSSION THREAD

**Discussion by RogueOJ** [itsToggle/plex_debrid#92] [Q&A]
Hi Toggle

just got this all set up with my vps however my ip seems to be blocked according to the real-debrid webpage, do you know of anyway around this,

ive tried with 
# export http_proxy=http://[username:pass@proxy-server-ip-or-dns-name]:[Port-Number]
and also https 

Any help much appreciated

Regards

**Comment by itsToggle** [comment]
Oh no! It seems that realdebrid is blocking the VPS IP address :( 

I don't use a shared VPS, so I've not encountered this issue. Using a VPS or VPN is generally not allowed on realdebrid - they have a whitelist of allowed public IPs, which can be found on https://real-debrid.com/vpn .
If you are using a shared VPS, your server IP is used by a lot of people - which is probably why realdebrid blocks it. Using a private VPS does not have this issue, as a private VPS has its own IPv4 address.
One way of getting around this would be to to run your VPS behind a whitelisted VPN. Another way would be to pay realdebrid for "remote traffic".

Also, always make sure to only ever use realdebrid from one location at a time. (Don't manually download or stream something on the realdebrid website from your home network, while you stream plex from your VPS. )

**Reply by RogueOJ** [reply]
ive tried installing a vpn on the vps but it fails also the vps company refuse to to let me run one as ive already spoke with them before coming here, ive used debrid for a long time and only ever use it from one location or ip addy :)

**Reply by itsToggle** [reply]
Damn, that sucks. I'll have another if I can find any more information :/

**Comment by RogueOJ** [comment]
thanks toggle ive been at this for hrs trying ways and means around it but come up short everytime :(

Many thanks

**Reply by itsToggle** [reply]
So no luck installing something like openVPN? I'm very sorry for the unreflected encouragement to use a VPS (as I said, I never expirienced any issues) -I will update the readme and name some of the problems people might encounter.

**Reply by RogueOJ** [reply]
no tried a few times that fails also, it lets you get as far as adding the config and as soon as you assign which server you want it to connect to it then fails and says the config file doesnt exist even though its there, hope that makes sense

**Reply by itsToggle** [reply]
Oh man..

**Reply by itsToggle** [reply]
I've updated the readme with a bit of a warning from shared VPS servers. Thanks for bringing this to my attention!

**Reply by RogueOJ** [reply]
Your more then welcome keep up the good work  (Y)

**Reply by RogueOJ** [reply]
while your here can you send me some info regarding netcup.de "if these are the ones you are using yourself"

Regards

**Reply by itsToggle** [reply]
sure thing: 
Ive owned a private VPS with them for a while and I'm very happy with their service. Their servers are located in Germany and Switzerland. 
The private VPS I own is 10€/m for unlimited 2.5Gbit/s traffic, a 4 core CPU, 8gb of RAM and a 120GB SSD. 
My Plex server, which I share with 5 other people, has never maxed out any of my specs. It can even transcode 4k HDR content.

**Reply by RogueOJ** [reply]
send me the link for the and your affil code ill get one later tonight it your sure it 100% works as i only got this one last night

**Reply by itsToggle** [reply]
This is the one I have:
https://www.netcup.eu/bestellen/produkt.php?produkt=2899

~Im paying extra for the larger bandwidth, I think during checkout you can add more custom features.~

You can either use my 5€ coupon from the readme:
`36nc16612918700`
or this one for 3 months off a 12 month plan: `2915nc16613261160`

I can only tell you that it works flawlessly for me.. it **probably** will for you as well. You could just get a one-month plan which you can cancel any time.

**Reply by RogueOJ** [reply]
thank you ill pick one up and let you know the outcome  :)

**Reply by itsToggle** [reply]
Great! The 2.5gbit/s bandwidth is the standard configuration. No need to add any custom features :)

**Comment by RogueOJ** [comment]
Quick update i cant purchase from the server comany as they want loads of identification to prove who i am, (which personally im not willing to do), i currently have access to 2 dedi servers with leaseweb who use ipv4 addys and they are blocked to :( guess it could be down to pure luck if the ip add is blocked or not from rd 

Going forward ill just use my windows machine, keep up the good work with the script it works well and saves a bunch of time 
In no means is this a rant was just an fyi in regards to servers and vps 

Regards
